### PR TITLE
fix athenz-sia.env

### PR DESCRIPTION
**Pull request by t4niwa**
# Description

- Specify default value for binary
- Provide the URL where the default value is set
- Adjust the order of environment variables to match [config.go](https://github.com/AthenZ/k8s-athenz-sia/blob/main/pkg/config/config.go)

## Type of change

- [ ] Bug fix
- [ ] New feature
- [ ] Refactoring (no functional changes, no api changes)
- [x] Non-code changes (update documentation, pipeline, etc.)

### Flags

- [ ] Breaks backward compatibility
- [ ] Requires a documentation update
- [ ] Has untestable code

## Related issue/PR

**Delete this section if there are no issues or pull requests that relate to this pull request.**
- Fixes #_issue_
- Closes #_PR_

---

## Checklist

- [ ] Followed the guidelines in the CONTRIBUTING document
- [ ] Added prefix `[skip ci]`/`[ci skip]`/`[no ci]`/`[skip actions]`/`[actions skip]` in the PR title if necessary
- [ ] Tested and linted the code
- [ ] Commented the code
- [ ] Made corresponding changes to the documentation
- [ ] Passed all pipeline checking

## Checklist for maintainer

- Use `Squash and merge`
- Double-confirm the merge message has prefix `[skip ci]`/`[ci skip]`/`[no ci]`/`[skip actions]`/`[actions skip]`
- Delete the branch after merge

### CODE REVIEW

**Review by mlajkim — DISMISSED** [review]
44 env variables with 44 settable envs
```sh
grep -v '^#' athenz-sia.env | wc -l
#     44
```

**Comment by t4niwa** [comment]
@mlajkim added in https://github.com/AthenZ/k8s-athenz-sia/pull/43/commits/e818143cfd58c52369021ef54d1614a4695419bc

File: athenz-sia.env [hunk 1]
```diff
@@ -1,8 +1,13 @@
+# Guidelines for managing the env file here:
+#   - If a default value is present, it must be provided with the URL where the default value is set
+#   - Follow the order of env variables as listed on https://github.com/AthenZ/k8s-athenz-sia/blob/main/pkg/config/config.go
+
```
Comment (mlajkim) [inline athenz-sia.env:4]:
```suggestion
#######################################################################################
# Guidelines for managing the env file here:
#   - If a default value is present, it must be provided with the URL where the default value is set
#   - Follow the order of env variables as listed on https://github.com/AthenZ/k8s-athenz-sia/blob/main/pkg/config/config.go
#######################################################################################
```
No blank line without # please

Comment (t4niwa) [reply athenz-sia.env]: fixed in https://github.com/AthenZ/k8s-athenz-sia/pull/43/commits/0234685a24ad480f71ba31f057151819ae91803e

**Review by mlajkim — APPROVED** [review]
OK

**Pull request merged**